### ISSUE TITLE
Ostrich Eggs

### ISSUE THREAD

**Issue by Gianna365** [DrZhark/mocreatures#212]
For so reason it's impossible to get a ostrich egg in survival the female ostrich won't lay any eggs I can only get them in creative. Please fix.

**Comment by CommandBlockGuru** [comment]
If I read the wiki correctly, I hear you need to find a female ostrich and a male ostrich near each other, then feed the female melon seeds, and in 1-2 mins an egg should pop out, and you wait till it hatches while staying near it, and it should be tamed.

**Comment by Gianna365** [comment]
I have done that multiple times and an egg never came.

**Comment by CommandBlockGuru** [comment]
Have you tried maybe going afk then coming back?

**Comment by Gianna365** [comment]
Yes but I would like to mention that it doesn't work only on 1.12.2

**Comment by CommandBlockGuru** [comment]
Hmm... Well, I'm sorry, but I don't know what to say about that. I don't work for the mod and I'm pretty sure the devs don't even check here, and I'd say to ask around on the wiki, but they don't allow you to comment for some reason. Sorry I couldn't help.
